ci-operator/config/Azure/ARO-HCP/Azure-ARO-HCP-main__periodic.yaml

```diff
@@ -69,57 +107,87 @@ tests:
     - ref: aro-hcp-provision-generate-resource-group-name
     test:
     - chain: aro-hcp-provision-create-functional-cluster
-- as: integration-e2e-parallel
+- as: prod-uksouth-e2e-parallel
   cron: 0 */4 * * *
   steps:
     env:
-      ARO_HCP_SUITE_NAME: integration/parallel
-      CUSTOMER_SUBSCRIPTION: ARO SRE Team - INT (EA Subscription 3)
+      ARO_HCP_SUITE_NAME: prod/parallel
+      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
       LOCATION: uksouth
     post:
     - ref: aro-hcp-provision-aro-hcp-gather-extra
     - chain: aro-hcp-provision-teardown-cluster
     test:
     - ref: aro-hcp-tests-run-aro-hcp-tests
-- as: stage-e2e-parallel
+- as: delete-expired-prod-uksouth-resource-groups
+  cron: 7,37 * * * *
+  steps:
+    env:
+      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
+    test:
+    - ref: aro-hcp-provision-delete-all-expired-resource-groups
+- as: create-aro-hcp-in-prod-brazilsouth
   cron: 0 */4 * * *
   steps:
     env:
-      ARO_HCP_SUITE_NAME: stage/parallel
-      CUSTOMER_SUBSCRIPTION: ARO HCP - STAGE testing (EA Subscription)
-      LOCATION: uksouth
+      CLUSTER_NAME: periodic-prod-brazilsouth-cluster
+      CUSTOMER_RESOURCE_GROUP_PREFIX: periodic-prod-brazilsouth-customer-rg
+      LOCATION: brazilsouth
+      SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
     post:
     - ref: aro-hcp-provision-aro-hcp-gather-extra
     - chain: aro-hcp-provision-teardown-cluster
+    pre:
+    - ref: aro-hcp-provision-generate-resource-group-name
     test:
-    - ref: aro-hcp-tests-run-aro-hcp-tests
-- as: prod-e2e-parallel
+    - chain: aro-hcp-provision-create-functional-cluster
+- as: prod-brazilsouth-e2e-parallel
   cron: 0 */4 * * *
   steps:
     env:
       ARO_HCP_SUITE_NAME: prod/parallel
       CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
-      LOCATION: uksouth
+      LOCATION: brazilsouth
     post:
     - ref: aro-hcp-provision-aro-hcp-gather-extra
     - chain: aro-hcp-provision-teardown-cluster
     test:
     - ref: aro-hcp-tests-run-aro-hcp-tests
-- as: delete-expired-integration-resource-groups
+- as: delete-expired-prod-brazilsouth-resource-groups
   cron: 7,37 * * * *
   steps:
     env:
-      CUSTOMER_SUBSCRIPTION: ARO SRE Team - INT (EA Subscription 3)
+      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
     test:
     - ref: aro-hcp-provision-delete-all-expired-resource-groups
-- as: delete-expired-stage-resource-groups
-  cron: 7,37 * * * *
+- as: create-aro-hcp-in-prod-centralindia
```

Comment:
remove job since e2e-parallel obviates the need

Reply:
Fixed with c96a514.